server/features/content_publish.feature

```diff
@@ -35,7 +35,8 @@ Feature: Content Publishing
       """
       And we publish "#archive._id#"
       Then we get OK response
-      And we get existing resource
+      When we get "/archive/#archive._id#"
```

Comment:
If this is needed, it means the returned doc is not having all the updates and the client side needs to refetch the item.